add new points to general writing tips (#673)
Codecademy/docs: documentation/content-standards.md

| `CatalogContent` | Slugs of Codecademy course and Path landing pages that relate to the entry. We're storing all of our slugs in the [catalog-content.md file](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md).<br /> - Please avoid linking to individual content items, because their URLs may change and some are gated by Pro membership.<br /> - We aim to include 2 slugs with every entry (1 free course and 1 Pro course/Path). | CatalogContent:<br /> - "introduction-to-javascript"<br /> - "paths/front-end-engineer-career-path" |

#### General Writing Tips

> - Be clear and concise

Reviewer: ```suggestion
```

I think this could be incorporated with the exiting tip beginning with "Brevity without sacrificing..."

- Avoid referencing information that isn't strictly related to the topic of the entry. As a rule, you want to assume as little pre-existing knowledge as possible.
- Avoid using first- and second-person pronouns (e.g. I, we, you) if possible.
- Brevity without sacrificing clarity. Make every word count.
- If the concept is hard, make it easy. If it's dry, make it fun. If it's simple, keep it simple.
- 90% of writing is rewriting.

> - Minimize the length of your sentence,and each one should communicate a point (paragraphs should be a maximum of 4-6 lines)

Reviewer: ```suggestion
- Minimize the length of your sentence,and each one should communicate a point (paragraphs should be a maximum of 4-6 lines).
```

> - Use white spaces and bullet points as much as you can (this makes words easier to read and aids comprehension)

Reviewer: ```suggestion
- Use white spaces, lists, and markdown tables wherever possible to improve readability and aid comprehension.
```

- Always explain an abbreviation the first time.
- Identify your target audience.
- Use punctuations properly.

> - Avoid using complex and non-contextual grammar.

Reviewer: ```suggestion
- Avoid using idioms as well as complex and non-contextual grammar.
```

> - Run your contents through writing tools (Grammarly, Prowriting Aid, Hemingway App) to check for grammar errors.

Reviewer: ```suggestion
- Run your contents through proofreading tools (e.g. Grammarly, Prowriting Aid, Hemingway App) to check for grammar errors.
```

- Re-read your contents carefully, considering every line written.

> - The title of your content should contain keywords for SEO purposes.

Reviewer: ```suggestion
- The title of your content should contain keywords for SEO purposes.

```

- Avoid referencing information that isn't strictly related to the topic of the entry. As a rule, you want to assume as little pre-existing knowledge as possible.
- Avoid using first- and second-person pronouns (e.g. I, we, you) if possible.
- Brevity without sacrificing clarity. Make every word count and be clear and concise.
- If the concept is hard, make it easy. If it's dry, make it fun. If it's simple, keep it simple.
- 90% of writing is rewriting.
- Minimize the length of your sentence,and each one should communicate a point (paragraphs should be a maximum of 4-6 lines).
- Use white spaces, lists, and markdown tables wherever possible to improve readability and aid comprehension.
- Always expand and explain an abbreviation the first time it is mentioned.
- Identify your target audience.
- Use punctuations properly.
- Avoid using idioms as well as complex and non-contextual grammar.
- Run your contents through proofreading tools (e.g. Grammarly, Prowriting Aid, Hemingway App) to check for grammar errors.
- Re-read your contents carefully, considering every line written.

> - The title of your content should contain keywords for SEO purposes.

Reviewer: ```suggestion
- The title of your content should contain keywords for SEO purposes.

```

If we add the empty line here that should allow the "verify" GitHub check to pass.

- Avoid referencing information that isn't strictly related to the topic of the entry. As a rule, you want to assume as little pre-existing knowledge as possible.
- Avoid using first- and second-person pronouns (e.g. I, we, you) if possible.
- Brevity without sacrificing clarity. Make every word count and be clear and concise.
- If the concept is hard, make it easy. If it's dry, make it fun. If it's simple, keep it simple.
- 90% of writing is rewriting.
- Minimize the length of your sentences,and each one should communicate a point (paragraphs should be a maximum of 4-6 lines).
- Use white spaces, lists, and markdown tables wherever possible to improve readability and aid comprehension.
- Always expand and explain an abbreviation the first time it is mentioned.
- Be aware of your target audience and write to their level.
- Use punctuation properly.
- Avoid using idioms as well as complex and non-contextual grammar.

> - Run your contents through proofreading tools (e.g. Grammarly, Prowriting Aid, Hemingway App) to check for grammar and spelling errors.

Reviewer 2: ```suggestion
- Run your content through proofreading tools (e.g. Grammarly, Prowriting Aid, Hemingway App) to check for grammar and spelling errors.
```

- Avoid referencing information that isn't strictly related to the topic of the entry. As a rule, you want to assume as little pre-existing knowledge as possible.
- Avoid using first- and second-person pronouns (e.g. I, we, you) if possible.
- Brevity without sacrificing clarity. Make every word count and be clear and concise.
- If the concept is hard, make it easy. If it's dry, make it fun. If it's simple, keep it simple.
- 90% of writing is rewriting.
-  Minimize the length of your sentences, and each one should communicate a point (paragraphs should be a maximum of 4-6 lines).
- Use white spaces, lists, and markdown tables wherever possible to improve readability and aid comprehension.
- Always expand and explain an abbreviation the first time it is mentioned.
- Be aware of your target audience and write to their level.
- Use punctuation properly.
- Avoid using idioms as well as complex and non-contextual grammar.
- Run your contents through proofreading tools (e.g. Grammarly, Prowriting Aid, Hemingway App) to check for grammar and spelling errors.
- Re-read your contents carefully, considering every line written.

> <<<<<<< HEAD

Reviewer 3: ```suggestion
```

- The description of your content should contain keywords for SEO purposes.
  

> =======

Reviewer 3: ```suggestion
```

> - The title of your content should contain keywords for SEO purposes.

Reviewer 3: ```suggestion
```

> >>>>>>> fb649c55aeab0bfb31037821228f7d3f20014d10

Reviewer 3: ```suggestion
```

- Avoid referencing information that isn't strictly related to the topic of the entry. As a rule, you want to assume as little pre-existing knowledge as possible.
- Avoid using first- and second-person pronouns (e.g. I, we, you) if possible.
- Brevity without sacrificing clarity. Make every word count and be clear and concise.
- If the concept is hard, make it easy. If it's dry, make it fun. If it's simple, keep it simple.
- 90% of writing is rewriting.

> - Minimize the length of your sentence,and each one should communicate a point (paragraphs should be a maximum of 4-6 lines).

Reviewer 3: ```suggestion
- Minimize the length of your sentences, and each one should communicate a point (paragraphs should be a maximum of 4-6 lines).
```

- Use white spaces, lists, and markdown tables wherever possible to improve readability and aid comprehension.
- Always expand and explain an abbreviation the first time it is mentioned.

> - Identify your target audience.

Reviewer 3: ```suggestion
- Be aware of your target audience and write to their level.
```

Prior verbiage implies the author is picking the target audience, Docs has its own intended audience that should be written to.

> - Use punctuations properly.

Reviewer 3: ```suggestion
- Use punctuation properly.
```

- Avoid using idioms as well as complex and non-contextual grammar.

> - Run your contents through proofreading tools (e.g. Grammarly, Prowriting Aid, Hemingway App) to check for grammar errors.

Reviewer 3: ```suggestion
- Run your contents through proofreading tools (e.g. Grammarly, Prowriting Aid, Hemingway App) to check for grammar and spelling errors.
```

- Re-read your contents carefully, considering every line written.

> - The title of your content should contain keywords for SEO purposes.

Reviewer 3: ```suggestion
- The description of your content should contain keywords for SEO purposes.
```

For Docs entries, the writer has little flexibility in titles. They generally have to be the class/function/method name. Descriptions are where they have some control.
